### PR TITLE
usswrite, regex check for illegal params in create

### PR DESCRIPTION
**Pull request by kostrja**

### CODE REVIEW

File: src/main/java/zowe/client/sdk/zosfiles/uss/methods/UssWrite.java [hunk 1]
```diff
@@ -9,5 +9,97 @@
...
+     * Replaces the content of a unix file with new content.
+     *
+     * @param destName destination name of where the file is located
+     * @param memberName  name of member to add new content
+     * @param content     new content
+     * @return http response object
+     * @throws Exception error processing request
+     * @author Frank Giordano
+     */
+    public Response write(String destName, String memberName, String content) throws Exception {
```
Comment (frankgiordano) [inline src/main/java/zowe/client/sdk/zosfiles/uss/methods/UssWrite.java:71]: this write method wont work for updating/writing to file. Correct? It seems to be a copy and paste from DSN package. We can remove it.

Comment (kostrja) [reply src/main/java/zowe/client/sdk/zosfiles/uss/methods/UssWrite.java:71]: yes guilty, i think i requested the pull too early

File: src/main/java/zowe/client/sdk/zosfiles/uss/methods/UssCreate.java [hunk 2]
```diff
@@ -97,9 +100,11 @@ public Response create(String destName, CreateParams params) throws Exception {
         ValidateUtils.checkNullParameter(params.getType() == null, "params type is null");
         ValidateUtils.checkNullParameter(params.getMode() == null, "params mode is null");
         ValidateUtils.checkNullParameter(destName == null, "destName is null");
+        ValidateUtils.checkIllegalParameter(params.getMode()
+                .matches("[r-][w-][w-][r-][w-][w-][r-][w-][w-]"), "params.mode must be of format 'rwxrwxrwx'");
```
Comment (frankgiordano) [inline src/main/java/zowe/client/sdk/zosfiles/uss/methods/UssCreate.java]: make the regex code performed in a helper private method in the class.. how about "x" in the match list. I dont see it.  You can write a method called isModeValidFormat(..) and return a boolean value.

File: src/main/java/zowe/client/sdk/zosfiles/uss/methods/UssCreate.java [hunk 3]
```diff
@@ -97,9 +100,11 @@ public Response create(String destName, CreateParams params) throws Exception {
...
 
-        final String url = "https://" + connection.getHost() + ":" + connection.getZosmfPort() +
-                ZosFilesConstants.RESOURCE + ZosFilesConstants.RES_USS_FILES + destName;
+        final String url = "https://" + connection.getHost() + ":" + connection.getZosmfPort()
+                + ZosFilesConstants.RESOURCE + ZosFilesConstants.RES_USS_FILES + destName;
```
Comment (frankgiordano) [inline src/main/java/zowe/client/sdk/zosfiles/uss/methods/UssCreate.java]: the project style is to have + at the end of the line and not at the start..

File: src/main/java/zowe/client/sdk/zosfiles/uss/methods/UssWrite.java (hunk 1 above)
Comment (frankgiordano) [reply src/main/java/zowe/client/sdk/zosfiles/uss/methods/UssWrite.java:71]: no problem. clean it up.. you can continue to checkin while PR is open and it gets updated.

**Comment by frankgiordano** [comment]
Closing PR request. Reclone mainline and work based on the new UssWrite scaffolding and create a new PR..

**Pull request closed without merging**